src/views/DogCard/DogDetail.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { deleteDog, fetchSingleDog } from '../../services/fetchdogs';
+import { useParams, useHistory } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import './DogDetail.css';
+
+export default function DogDetail({ currentUser }) {
+  const params = useParams();
+  const id = params.id;
+  const [dogData, setDogData] = useState([]);
+  const history = useHistory('');
+  
+ 
+
+  useEffect(() => {
+    const fetchDogData = async () => {
+      const data = await fetchSingleDog(id);
+      setDogData(data);
+    };
+    
+    fetchDogData();
+  }, [id]);
+
+  const removeDog = async () => {
+    await deleteDog(id); 
+    history.push(`/`);
+  };
+
+  if (!dogData) return <h1>...Walking Doggos</h1> ;
+
+  return (
+    <div className='test'>
+      {currentUser && (
+        <>
+          <Link exact to = {`/dogs/${dogData.id}/edit`}>
+            <button className='Edit'>Edit info</button>
+          </Link>
+          <button onClick={removeDog}>Delete</button>
+        </>
```

Comment:
Great conditional rendering here to show the edit and delete buttons for logged in users.